neurom/fst/_mm.py

```diff
@@ -101,11 +101,6 @@ def map_sections(fun, nrn, neurite_type=NeuriteType.all):
                 for s in i_chain2(nrn.neurites, tree_filter=is_type(neurite_type)))
 
 
-def section_lengths(nrn, neurite_type=NeuriteType.all):
-    '''section lengths'''
-    return map_sections(mm.path_distance, nrn, neurite_type)
```

Comment:
OK, you are definitely right. Here is a more concrete definition. 

*Neuroanatomical Tract-Tracing: Molecules, Neurons, and Systems*,  by Laszlo Zaborszky, Floris G. Wouterlood, José Luis Lanciego 

> The dendritic tortuosity gives a ratio between the actual length of a dendritic segment (our section) and the distance between its endpoints